Jenkinsfile

```
@@ -89,7 +89,7 @@ node('docker&&linux') {
              */
             stage('Verify Container') {
                 container.withRun("--link ${c.id}:nginx -e DATA_FILE_URL=http://nginx/plugins.json.gzip") { api ->
-                    docker.image('cirrusci/wget:latest').inside("--link ${api.id}:api") {
+                    docker.image('bash:latest').inside("--link ${api.id}:api") {
```

latest bites us too often
```suggestion
                    docker.image('bash@sha256:95599adce80c5f938a8445eca59a8ac4e380f75b0e3e21971b37099c0c54a187').inside("--link ${api.id}:api") {
```